test/integration/targets/uri/tasks/use_netrc.yml

```diff
@@ -0,0 +1,55 @@
+- name: Write out netrc
+  copy:
+    dest: "{{ remote_tmp_dir }}/netrc"
+    content: |
+      machine httpbin.org
+      login foo
+      password bar
+
+- name: Read netrc
+  debug: var=item 
+  with_file:
+    - "{{ remote_tmp_dir }}/netrc"
```

Comment:
Lookups cannot consult things on the target.  This will need converted to a slurp task + debug task if you want to keep this.

Reply:
Decided to remove it since it's not the main test here.